components/applications-service/pkg/server/server.go

```diff
@@ -260,6 +260,26 @@ func (app *ApplicationsServer) GetServicesStats(c context.Context,
 	}, nil
 }
 
+// GetDisconnectedServices returns a list of disconnected services
+func (app *ApplicationsServer) GetDisconnectedServices(c context.Context,
```

Comment:
[convention] let's use `ctx` here please:
```suggestion
func (app *ApplicationsServer) GetDisconnectedServices(ctx context.Context,
```

Reply:
Ok

Reply:
Will change it in a different PR since all of them are the same 😅

Reply:
Created PR to change this https://github.com/chef/automate/pull/889